bigbluebutton-html5/imports/ui/components/chat/chat-dropdown/component.jsx

```diff
@@ -97,7 +97,7 @@ class ChatDropdown extends PureComponent {
             `data: ${mimeType} ;charset=utf-8,
             ${encodeURIComponent(ChatService.exportChat(ChatService.getPublicGroupMessages()))}`,
           );
-          link.click();
+          link.dispatchEvent(new MouseEvent('click', { bubbles: true, cancelable: true, view: window }));
```

Comment:
Does this work on Chrome too, Safari, Edge, mobile*?
I thought there would be some browser checking condition in the solution. Would be glad if it's all standard now :)

Reply:
https://javascript.info/dispatch-events#event-constructor
![screenshot from 2019-02-15 10-31-55](https://user-images.githubusercontent.com/6312397/52866581-f3227100-310c-11e9-8d62-7fcc7cb2fed0.png)